<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameScene</class>
 <widget class="QWidget" name="GameScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="gameGraphicsView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>400</width>
     <height>800</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="nextBlockGraphicsView">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>100</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="scoreWidget" native="true">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>335</y>
     <width>201</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="scoreLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>分数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="scoreLCD">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>840</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restart_btn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>770</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>700</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>存档</string>
   </property>
  </widget>
  <widget class="QLabel" name="pauseLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>420</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>PAUSE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
